docs/docs/gatsby-link.md

```diff
@@ -8,39 +8,81 @@ Gatsby's `<Link>` component enables linking to internal pages as well as a power
 
 The component is a wrapper around [@reach/router's Link component](https://reach.tech/router/api/Link) that adds useful enhancements specific to Gatsby. All props are passed through to @reach/router's `Link` component.
 
-## How to use
+## How to use Gatsby Link
 
-In JavaScript:
+<iframe class="egghead-video" width=600 height=348 src="https://egghead.io/lessons/egghead-why-and-how-to-use-gatsby-s-link-component/embed" />
```

Comment:
Can you add titles to all video iframes? 

That would make a good automated test so the computer can remind you instead of me 😄

Reply:
Done!

Reply:
(Also now that I know this is a best practice I'll just do it moving forward. 😄)